pass/pass_linux.go

```diff
@@ -95,12 +79,38 @@ func runPass(stdinContent string, args ...string) (string, error) {
 	return string(result), nil
 }
 
+func isInitialized() bool {
+	if passInitialized {
```

Comment:
If an error occurs during the first pass of `isInitialized`, then `passInitialized` won't be true, and then we will go through `isInitialized` again, and maybe again etc... I think you should return an error if it fails and let the caller prevent from calling `isInitialized` again.

Reply:
@silvin-lubecki and in this case you just disable this credentials storing method?

Reply:
Why not? If there is an error, it means this credentials storing method is not available no? We could add a retry pattern here, but I don't think it would really help...